RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/RecoverWalletWithoutProfile/Children/Completed/RecoverWalletControlWithBDFSComplete+View.swift

```diff
@@ -20,45 +11,28 @@ public extension RecoverWalletControlWithBDFSComplete {
 		}
 
 		public var body: some SwiftUI.View {
-			WithViewStore(store, observe: \.viewState, send: { .view($0) }) { _ in
-				VStack(alignment: .leading, spacing: .medium1) {
-					Text("Recovery completed")
-						.textStyle(.sheetTitle)
+			VStack(alignment: .leading, spacing: .medium1) {
+				// FIXME: Strings
+				Text("Recovery completed")
+					.textStyle(.sheetTitle)
+					.multilineTextAlignment(.center)
 
-					Text("Accounts discovered in the scan have been added to your wallet.\n\nIf you have any Olympia or “Legacy” Accounts to import - or any Accounts using a Ledger hardware wallet device - please use the **Account Recovery Scan** option in your Radix Wallet settings under **Account Security**.")
-						.textStyle(.body1Regular)
+				// FIXME: Strings
+				Text("Accounts discovered in the scan have been added to your wallet.\n\nIf you have any Olympia or “Legacy” Accounts to import - or any Accounts using a Ledger hardware wallet device - please use the **Account Recovery Scan** option in your Radix Wallet settings under **Account Security**.")
+					.multilineTextAlignment(.leading)
 
-					Spacer(minLength: 0)
-				}
-				.padding()
-				.footer {
-					Button("Continue") {
-						store.send(.view(.continueTapped))
-					}
-					.buttonStyle(.primaryRectangular)
+				Spacer(minLength: 0)
+			}
+			.textStyle(.body1Regular)
+			.foregroundColor(.app.gray1)
+			.padding()
+			.footer {
+				// FIXME: Strings
```

Comment:
If you put the FIXME on the same line as the string, it's visible in the search results in xcode. Just simplifies the process a bit.